File: src/rotatearray/naive/Driver.java
package rotatearray.naive;

import java.util.Arrays;

public class Driver {
    public static void main(String[] args) {
              // Input: nums = [1,2,3,4,5,6,7], k = 3
        int[] nums = {1,2,3,4,5,6,7};
        int k = 5;
        Solution sol = new Solution();
        sol.rotate(nums, k);
        System.out.println(Arrays.toString(nums));

    }
}

File: rotate-array/cpp/Driver.cpp
#include "Solution.h"
#include <vector>
#include <iostream>

int main() {
    std::vector<int> nums = {1,2};
    int k = 2;
    Solution sol;
    sol.rotate(nums, k);
    std::cout << "The rotated array " << k << "spots to the right is: ";
    for (int num : nums) {
        std::cout << num;
    }
    return 0;
}